test/k8s-integration/cluster.go

```diff
@@ -193,5 +194,14 @@ func getGKEKubeTestArgs() ([]string, error) {
 		fmt.Sprintf("--gcp-project=%s", project[:len(project)-1]),
 	}
 
+	out, err := exec.Command("which", "kubectl").Output()
```

Comment:
where does kubectl come from? The test container image binary?  kubectl has interesting compatibility behavior with Kubernetes versions.  The kubectl version from the container image may get stale.  How do our regular gke tests get kubectl?

Reply:
It comes from the test image. There really isnt a way to make sure kubectl version is compatible with GKE version unless we build the exact version of kubectl for our GKE cluster version. We can get close by using kubectl built in the testing directory, so at least as long as were using the latest k8s e2e tests then kubectl version >= gke version. 

I updated the code to do this.

Reply:
Is this what normal gke tests too? We should follow closely what is done for the regular tests.

Reply:
Do you mean other tests not owned by sig storage that run on GKE? They do t try to boot their own gke cluster. They let kubetest do the cluster up. And thus they do t have to worry about kubectl as kubetest will manage that for them.